learn/app-development/core/core-overview.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Core Overview"
+id: "core-overview"
+sidebar_label: "core overview"
+---
+---
+
+Core Project can be defined as a base application that contains basic functionalities that can be later used to create child applications on top of the base core application. It is similar to Project shell that is already supported in WaveMaker but offers some extended features like easy upgrade of the core application at any point of the development of the child application and so on.
+
+To understand the differences between Project shell and Core project
+
+|Project Shell|Core Project|
```

Comment:
We should not emphasize this comparision so prominently. We can move this to the bottom as a question

How is this different from a project shell?